src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -91,16 +93,14 @@ private IEnumerable<LibraryExport> ExportLibraries(Func<LibraryDescription, bool
                     }
                 }
 
+                // Source references are not transitive
```

Comment:
We should probably to this for analyzers too until we implement the new nuget transitive flowing of assets.